### ISSUE TITLE
Do you have the kth_action dataset?

### ISSUE THREAD

**Issue by toddwyl** [thuml/predrnn-pytorch#3]
Do you have the kth_action dataset ?
I use the kth_action dataset from the official website, and preprocess the video to frames.
But I cannot get **the training set of 108,717 and test set of 4,086 sequences** as the paper mention. 
I get like below: 
```
training set:
there are 183861 pictures
there are 30379 sequences
test set:
there are 105855 pictures
there are 17145 sequences
```
Can u give me the preprocess method or the preprocessed dataset ？

**Comment by toddwyl** [comment]
@wyb15 And I found the moving mnist dataset have no MNIST-3 dataset, could u offer me the dataset?
I could not get the kth test sequences until now. But I get the similar training set which have 115024 sequences. Could u offer me the dataset kth and MNIST-3?
Thanks a lot!

**Comment by GuptaSonam** [comment]
Hi, 
were you able to train it on KTH data successfully? I tried but I am not able to get the results which are reported in the paper.

**Comment by toddwyl** [comment]
@GuptaSonam I cannot get the correct data and cannot reproduce the results.

**Comment by GuptaSonam** [comment]
@wyb15 Would it be possible for you to share the preprocessed KTH data? I trimmed the videos manually but the results that I am getting are far from the values reported in the paper.
Thanks

**Comment by jlzcode** [comment]
@ManWingloeng I've got the KTH dataset. I can send it to you, please let me know your email address. 
Btw, is the training code incompletely for training KTH dataset?

**Comment by GuptaSonam** [comment]
@ml18j47z : Thanks. My email id is cs18d005@cse.iitm.ac.in I would be glad to get the processed KTH  data. 
I wrote the training script myself similar to the mnist training script provided in the repository.  I can share it with you if you want.

**Comment by jlzcode** [comment]
@GuptaSonam  I have sent the dataset to you. Please check it. Can you share this part of the code with me? My email is ml18j47z@leeds.ac.uk.Thanks a lot!

**Comment by toddwyl** [comment]
@ml18j47z My email id is yongliangwen@foxmail.com . I use the code [e3d_lstm/src/data_provider/kth_action.py](https://github.com/google/e3d_lstm/blob/master/src/data_provider/kth_action.py) in `data_provider`, but you should know e3d_lstm have some bugs in `e3d_lstm` model. 
Thanks a lot.

**Comment by wuhaixu2016** [comment]
Hi, we upload the dataset to Tsinghua cloud, you can find here:
https://cloud.tsinghua.edu.cn/d/7d19372a621a4952b738/
